src/pages/ReimbursementAccount/IdentityForm.js

```diff
@@ -133,7 +133,7 @@ const IdentityForm = (props) => {
                 errorText={props.errors.ssnLast4 ? props.translate('bankAccount.error.ssnLast4') : ''}
                 maxLength={CONST.BANK_ACCOUNT.MAX_LENGTH.SSN}
             />
-            {props.manualAddress ? (
+            {props.values.manualAddress ? (
```

Comment:
Do the other props need to be updated as well? For example, `value={props.city}` is still used even though `city` is inside `values` in the propTypes definition.

Reply:
Uh yeah good thinking, probably. I was too focused on fixing the issue and didn't think to check the rest,  I'll check now.